.github/workflows/webviz-subsurface.yml

```diff
@@ -52,6 +52,8 @@ jobs:
           pip freeze | grep -vw "pip" | xargs pip uninstall -y
         fi
         pip install 'pandas==${{ matrix.pandas-version }}'
+        pip install dash
+        pip install -i https://test.pypi.org/simple/ webviz-subsurface-components 
```

Comment:
`webviz-subsurface-components >= 0.0.27` on official PyPI now contains the new `LeafletMap` (= "LayeredMap v2") component.